src/shared/logger.ts

```diff
@@ -90,10 +90,10 @@ function getColor(elem, attr) {
   let color;
 
   do {
-    color = kendo.jQuery.css(elem, attr);
+    color = (<any>kendo.jQuery).css(elem, attr);
```

Comment:
jquery doesn't seem to expose a css property in their typescript definitions (probably because it is a private function)